### PR TITLE
Pan os address and applications groups crud

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-914)

## Description
Add CRUD (create, delete, update, list) operations for application-group + edit an address object + enhancement to address creation

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 1]
```diff
@@ -3774,6 +3774,8 @@ script:
       name: ip_netmask
     - description: The IP range of the new address IP. For example, 10.10.10.0-10.10.10.255
       name: ip_range
+    - description: The IP wildcard of the new address. For example, 10.20.1.0/0.0.248.255
```
Comment (JudahSchwartz) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:3777]: where does 0.0.248.255 come from? Doesnt look like a valid subnet to me. are the 0s wildcard numbers here?

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 2]
```diff
@@ -1317,14 +1318,18 @@ def panorama_create_address_command(args: dict):
...
-        raise Exception('Please specify exactly one of the following: fqdn, ip_netmask, ip_range.')
+    # make sure only one of fqdn/ip_netmask/ip_range/ip_wildcard was provided.
 
-    if (fqdn and ip_netmask) or (fqdn and ip_range) or (ip_netmask and ip_range):
-        raise Exception('Please specify exactly one of the following: fqdn, ip_netmask, ip_range.')
+    if sum(
+        map(bool, [(fqdn is not None), (ip_netmask is not None), (ip_range is not None), (ip_wildcard is not None)])
+    ) != 1:
+        raise DemistoException(
+            f'Please specify exactly one of the following arguments: fqdn, ip_netmask, ip_range, ip_wildcard.'
```
Comment (JudahSchwartz) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:1329]: you cant have a range with a subnet? eg 10.0.15.0 - 10.0.15.15  / 255.255.255.0

**Review by JudahSchwartz — APPROVED** [review]
Beast!
Just a couple questions

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml (hunk 1 above)
Comment (GuyAfik) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.yml:3777]: its taken directly from their documentation
image

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 2 above)
Comment (GuyAfik) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:1329]: image

you can't have a range with a subnet, only one of them is allowed

**Pull request merged**